### ISSUE TITLE
ARSceneview / Sceneview Camera differences

### ISSUE THREAD

**Issue by kunal-wayfair** [google-ar/sceneform-android-sdk#421]
I have two sessions:
1. ARSceneView
2. Sceneview
I am trying to save a pose of an object in a ARSceneView for a given frame and re place the same object in a (non AR) Sceneview.

For that, I save all the world poses from ARSceneView, in my Sceneview,  and place the camera and object at the same saved world poses.

My replicated pose of the object is almost correct but the output of what I see is not exact of what I saw / captured in the ARsceneview. The re placement in the SceneView  is off by some amount.

I can think of a few reasons for these:
Either the Camera Intrinsics / Extrinsics are different between the two AR / Non AR SceneViews.
Or, The world origins are differently placed w.r.t. the Screen in both the SceneViews.

In either case I cannot figure out that cause of the offset in re placement.
Any help /  suggestions  would be appreciated.

TLDR: I am trying to convert world pose at a given frame from ARSceneView to a equivalent world pose in SceneView.

**Comment by khonakr** [comment]
So the FOV I calculated are correct.
AR = ~64 Degrees SceneVIew = 90 Degrees.

I have got the scaling to work (1.53  = Tan 45 / Tan 32) But I am having a hard time calculating the pose of the object w.r.t the different FOVs.


Are there going to be plans to support letting us change SceneView's FOV?

**Comment by khonakr** [comment]
I finally figured out. For future reference if anyone is stuck, setting Projection Matrix changes camera's FOV. I had to apply ARSceneView's Projection Matrix to SceneView to get the correct FOV.
